source/Octopus.Versioning.Tests/VersionTests.cs

```diff
@@ -84,6 +84,23 @@ public void TestSemVerEquivalents()
                 "Another `.0` after patch isn't counted");
         }
 
+        [Test]
+        public void TestSingleDigit()
+        {
+            Assert.AreEqual(
+                VersionFactory.CreateSemanticVersion("0001"),
```

Comment:
[Semver 2.0.0 specs](https://semver.org/#spec-item-2) state that major/minor/patch cannot have leading 0's.
Is this a valid use case?

Reply:
Nevermind, these are valid "octopus semvers".